src/pages/Story.js

```diff
@@ -205,6 +237,14 @@ const Story = (props) => {
                         <Button className='share-story' onClick={copy}>
                           <EOS_CONTENT_COPY className='eos-icons' />
                         </Button>
+                        {permissions.includes('Delete Story') && (
```

Comment:
Can you also add some documentation in an .md file stating that a permission with this name "Delete Story" has to be configured on the backend (Strapi) and assigned to a role?

Reply:
Sure